sourcemash/api/feeds.py

```diff
@@ -10,6 +10,13 @@
 
 from sourcemash.models import Feed, UserItem, Item
 
+import os
+from rq import Queue
+from worker import create_worker
+from worker.scraper import scrape_feed_articles
+
+conn = create_worker(os.environ.get("APP_CONFIG_FILE") or "development")
```

Comment:
What would the process of moving this to a config file look like? Like what lines would go where? I'll do it but I'm pulling a bit of a blank on where to put things now